lib/chef_zero/endpoints/node_identifiers_endpoint.rb

```diff
@@ -0,0 +1,22 @@
+require 'ffi_yajl'
+require 'chef_zero/rest_base'
+require 'uuidtools'
+
+module ChefZero
+  module Endpoints
+    # /organizations/NAME/nodes/NAME/_identifiers
+    class NodeIdentifiersEndpoint < RestBase
```

Comment:
What is the documentation on how this endpoint is supposed to work?  Did you just look at the server code?